test/post_processor/test_dual_from_empirical_frequencies.py

```diff
@@ -47,8 +46,7 @@ def setUp(self) -> None:
         #   qc = random_circuit(num_qubits=2, depth=1, measure=False, seed=30)
         # for qiskit==1.1.1
         with open("test/post_processor/random_circuit_qubits=2_depth=1_seed=30.qpy", "rb") as file:
-            qc = qpy.load(file)
-        qc = random_circuit(num_qubits=2, depth=1, measure=False, seed=self.RNG_SEED)
+            qc = qpy.load(file)[0]
```

Comment:
Oops I must have overlooked that :sweat_smile:

Reply:
It was the last commit I went a bit fast as well 😅